sre-recipes/recipes/ratings_freshness_recipe/ratings_freshness_recipe.py

```diff
@@ -0,0 +1,134 @@
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -*- coding: utf-8 -*-
+"""
+This module contains the implementation of rating_freshness recipe
+"""
+
+import logging
+import subprocess
+from recipe import Recipe
+
+
+class RatingsFreshnessRecipe(Recipe):
+    """
+    This class implements 'rating_freshness' which purposefully
+    stops recollect API calls to rating service.
+    """
+
+    name = "rating_freshness"
+
+    def get_name(self):
+        return self.name
+
+    def is_active(self):
+        return True
+
+    def break_service(self):
+        """
+        Pause Cloud scheduler job to stop calls to recollect API of
+        the rating service
+        """
+        logging.info("Pausing scheduled job of the rating service")
+        print("Breaking rating operations...")
+        project_id = Recipe._get_project_id()
+        if not project_id:
+            print("Failed: cannot find project id.")
+            logging.error("Failed pausing scheduled job: no project id.")
+            exit(1)
+        # hide error output to avoid additional hints :)
+        pause_command = "gcloud scheduler jobs pause ratingservice-recollect-job --project {pid} 2> /dev/null".format(
+            pid=project_id
+        )
+        Recipe._run_command(pause_command)
+        print("...done")
+        logging.info("Scheduled job of the rating service paused")
+
+    def restore_service(self):
+        """
+        Resume Cloud scheduler job to restore calls to recollect API of
+        the rating service
+        """
+        logging.info("Resuming scheduled job of the rating service")
+        print("Restoring rating operations...")
+        project_id = Recipe._get_project_id()
+        if not project_id:
+            print("Failed: cannot find project id.")
+            logging.error("Failed pausing scheduled job: no project id.")
+            exit(1)
+        # hide error output to avoid additional hints :)
+        resume_command = "gcloud scheduler jobs resume ratingservice-recollect-job --project {pid} 2> /dev/null".format(
+            pid=project_id
+        )
+        Recipe._run_command(resume_command)
+        print("...done")
+        logging.info("Scheduled job of the rating service resumed")
+
+    def hint(self):
+        """
+        Provides a hint about the root cause of the issue
+        """
+        project_id = Recipe._get_project_id()
+        print(
+            "Try to change a rating for a shop's items one or more times.",
+            "Try to check SLO Monitoring and look for SLOs that consume error budget.",
+            "For SLO Monitoring open https://pantheon.corp.google.com/monitoring/services?project={pid}".format(
+                pid=project_id
```

Comment:
I saw the rating field disappear after submitting a rating. Is that what was expected?

Reply:
yes, it is a designed behavior. it is intended as a simulation of the one time rating submission.
BTW, it is not part of this change :slightly_smiling_face: